jwst/resample/resample.py

```diff
@@ -237,10 +236,8 @@ def resample_many_to_one(self):
             )
         )
 
-        # TODO: The following two methods and calls should be moved upstream to
-        # ResampleStep and ResampleSpecStep respectively
-        if isinstance(output_model, datamodels.ImageModel):
-            self.update_fits_wcs(output_model)
+        # TODO: The following method and call should be moved upstream to
```

Comment:
Should the `self.update_slit_metadata` method be moved up into the ResampleSpecStep class, like you are did for `self.update_fits_wcs`?  Not doing so would seem to introduce an inconsistency between the imaging and slit resample operations.

Reply:
@stsci-hack I was not sure whether I should do that for this ticket or open a new ticket for spectroscopy and do it then. Given your input, I'll do it right away.